otx/algorithms/action/configs/classification/base/base_classification_dynamic.py

```diff
@@ -0,0 +1,23 @@
+"""Base Action classification mmdeply cfg."""
+
+ir_config = dict(
+    type="onnx",
+    export_params=True,
+    keep_initializers_as_inputs=False,
+    opset_version=11,
+    save_file="end2end.onnx",
+    input_names=["data"],
+    output_names=["logits"],
+    input_shape=None,
+    optimize=True,
```

Comment:
```suggestion
    optimize=False,
```
`optimize` of `mmedploy` messes up NNCF graph. It would be better to be `False`.

Reply:
Thank you for notice, it is modified